### DISCUSSION TITLE
Overparametrisation of Models

### DISCUSSION THREAD

**Discussion by faaa6** [RePsychLing/SMLP2023#26] [Q&A]
Are there any other approaches to assess overparametrisation of models besides cheickng the models fit with AIC, BIC and LRTs and checking models for being singular?

**Comment by kliegl** [comment]
We also check whether **principal components (PCs)** of the random effects account for unique variance (MixedModels.PCA(fm)  for MixedModel.jl in Julia; summary(rePCA(fm)) for lme4 in R) . Usually, if you recover all the information with fewer PCs than there are variance components, the model is overparameterized. And:  LRT, AIC, and BIC are used not only once, but repeatedly in the context of strategies to reduce the complexity of the random effect structure. Most of our strategies are described in Bates et al. (2015; 2nd edition: 2018; https://arxiv.org/abs/1506.04967 ).  Compulsively minded persons will find a collection of exercises in complexity reduction for different contrasts here:  https://rpubs.com/Reinhold .
